index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Website</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+  <main>
```

Comment:
- [ ] This image is currently missing from your design. Kindly add it.

figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/227382524-2b143f61-5ea2-4613-a4e5-13012678bfd7.png)

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/227382579-e30aec7e-8232-4f9d-b98f-cd694d513356.png)